Makefile.python.am

```diff
@@ -20,7 +19,7 @@ all-local: $(top_builddir)/ipasetup.py
 		    --build-base "$(abs_builddir)/build"
 
 install-exec-local: $(top_builddir)/ipasetup.py
-	if [ "x$(pkginstall)" = "xtrue" ]; then \
+	if [ "x$(pkginstall)" != "xfalse" ]; then \
```

Comment:
> Default to pkginstall=true without duplicated definitions

automake was complaining about duplicated definition. I dropped ```pkginstall=true``` from ```Makefile.python.am``` and only set it in special ```Makefile.am```